jobs/build/sweep/Jenkinsfile

```diff
@@ -57,24 +57,31 @@ node {
     def kind = (major == '4') ? 'image' : 'rpm'
 
     stage("Repair bug state") {
-        try {
-            currentBuild.description += "* Moving attached bugs in MODIFIED state to ON_QA...\n"
-            buildlib.elliott "--group=openshift-${version} repair-bugs --use-default-advisory ${kind} --auto" 
-        } catch (elliottErr) {
-            currentBuild.description = "Error repairing:\n${elliottErr}"
-            throw elliottErr
+        retry (3) {
+            try {
+                currentBuild.description += "* Moving attached bugs in MODIFIED state to ON_QA...\n"
```

Comment:
This can:
- end up in build log
- before the `retry` block, so it only shows once